data/json/items/vehicle/engine.json

```diff
@@ -51,24 +51,24 @@
     "type": "ENGINE",
     "name": { "str": "1-cylinder engine" },
     "description": "A single-cylinder 4-stroke combustion engine.",
```

Comment:
Please add a comment to each engine what you've based it off so that future contributors can know how you've come to this conclusion and how to see that the sizes are correct.

Reply:
Since comments can't be added to JSON files, should I just update the ingame descriptions to better show engine tier positions?

Reply:
Gas engine recipes have descriptions - Ive tweaked those slightly to make tiers clear.  Motor vehicle parts have displacement inserted to show their tiers, and a json only comment to show what size vehicle these motors go in.

Electric motor descriptions, I'll put in the other PR as they need a bit more work